app/views/editions/areas.js.erb

```diff
@@ -1,3 +1,3 @@
 var areas = <%= raw @areas.map { |a|
-  { id: a.slug, text: a.name, country: a.country_name, type: a.type }
+  { id: "#{a.slug};#{a.codes["gss"]}", text: a.name, country: a.country_name, type: a.type }
```

Comment:
I agree that putting both values in the form isn't pretty, but since it's only going to be here temporarily while we migrate to using GSS codes I think it's OK. I'm happy with doing it in this template for the same reason.